src/commands/create_command.py

```diff
@@ -1,32 +1,24 @@
-import json, os
+import os
 from commands.abstract_command import *
-from commands.keys import Keys
-from output.exceptions import *
+from model.database import *
 
 
 class CreateCommand(AbstractCommand):
-    def __init__(self, schema_file):
-        self.validate(schema_file, Keys.SCHEMA_PATH)
-        schema_file = os.path.join(Keys.SCHEMA_PATH, schema_file)
-        with open(schema_file, 'r') as file:
-            self.data = json.load(file)
-        self.check_database()
-        self.path = os.path.join(Keys.DATABASE_PATH, self.data[Keys.DATABASE])
+    def __init__(self, schema_name):
+        self.validate(schema_name, Keys.SCHEMA_PATH)
+        self.schema_name = schema_name
 
     def execute(self):
-        os.makedirs(self.path, exist_ok=True)
-        for table in self.data[Keys.TABLES]:
-            t_path = os.path.join(self.path, table[Keys.NAME])
-            os.makedirs(t_path, exist_ok=True)
+        self.__create_database__()
 
     @staticmethod
-    def validate(schema_file, schema_dir):
-        if schema_file is None or schema_file == "" or schema_file == " ":
+    def validate(schema_name, schema_dir):
+        if schema_name is None or schema_name == "" or schema_name == " ":
```

Comment:
Search how to check string is null or empty in python

Reply:
what do you mean by schema_name here ? 

`CreateCommand` should validate the schema file exists + convert to json.